### ISSUE TITLE
hot to use export_api?

### ISSUE THREAD

**Issue by Fitz-Wen** [DHTMLX/gantt#96]
### install [dhtmlx-gantt](https://www.npmjs.com/package/dhtmlx-gantt) 
```
gantt.plugins({
    export_api: true
 })
```
### use `gantt.exportToPDF`
```
gantt.exportToPDF is not a function
```
![image](https://github.com/DHTMLX/gantt/assets/26396034/6d3d791a-8c82-41f2-8477-13c7ba43fbf4)

**Comment by datastructurey** [comment]
I also encountered, I found that the plugin package does not seem to have this parameter to use

**Comment by gearcoded** [comment]
@Fitz-Wen and @datastructurey, the documentation assumes you are running the latest Gantt version. Starting from the 8.0 version, you need to use the `plugins` method.
In earlier versions, you need to connect the export's `api.js` file:
https://export.dhtmlx.com/gantt/api.js

Here is the migration article:
https://docs.dhtmlx.com/gantt/migrating.html#7180:~:text=resource%20assignments%20section.-,Export%20service,-From%20v8.0%2C%20the

Please note, that the file is not updated, so if there are some bugs related to that file, you won't get updates.

I will add a task for the technical writer to add a note for the earlier versions.

**Comment by gearcoded** [comment]
Recently, it was discovered that the import and export functionality is not added via the `plugins` method in the 8.0.0-8.0.5 versions. It will be fixed in the newer versions, but I cannot give you any ETA.

**Comment by gearcoded** [comment]
*@Fitz-Wen* and *@datastructurey*,
The issue should be fixed in the 8.0.6 version:
https://docs.dhtmlx.com/gantt/whatsnew.html#:~:text=The%20Export%20API%20client%20is%20now%20integrated%20into%20the%20GPL%20version%20of%20Gantt%2C%20previously%20it%20has%20been%20included%20only%20in%20PRO%20versions
